### PR TITLE
Update ipmitool from 1.8.13 to 1.8.17

### PR DESCRIPTION
**Pull request by helloarys**
@fub2 @xiar

### CODE REVIEW

**Comment by xiar** [comment]
Can one of the admins verify this patch?

File: vcompute/dell_r730xd/data/dell_r730xd.emu [hunk 1]
```diff
@@ -12,17 +12,17 @@ sel_enable 0x20 1000 0x0a
 # Add DELL R730XD FRUs
```
Comment (fub2) [inline vcompute/dell_r730xd/data/dell_r730xd.emu]: ??? wrongly copy/paste?

Comment (helloarys) [inline vcompute/dell_r730xd/data/dell_r730xd.emu]: Wrong about what？
I have checked the dell_r630's emu file, and it seems like this.
Do you mean it should be 'Dell R730xd FRUs'?
@fub2

**Comment by PaynePei** [comment]
retest this please

**Comment by PaynePei** [comment]
retest this please

**Comment by fub2** [comment]
@helloarys I mean PR title is update IPMItool infraSIM is carrying; but there's content changing emulation data for a virtual node. it is leading confusion.

**Comment by PaynePei** [comment]
retest this please

**Comment by PaynePei** [comment]
retest this please

**Pull request closed without merging**